xyzservices/data/providers.json

```
@@ -2362,6 +2369,34 @@
             "variant": "MODIS_Terra_Chlorophyll_A",
             "opacity": 0.75,
             "name": "NASAGIBS.ModisTerraChlorophyll"
+        },
+        "ModisTerraBands721CR": {
+            "url": "https://gibs.earthdata.nasa.gov/wmts/epsg3857/best/MODIS_Terra_CorrectedReflectance_Bands721/default/{time}/GoogleMapsCompatible_Level9/{z}/{y}/{x}.jpg",
+            "max_zoom": 9,
+            "attribution": "Imagery provided by services from the Global Imagery Browse Services (GIBS), operated by the NASA/GSFC/Earth Science Data and Information System (<a href=\"https://earthdata.nasa.gov\">ESDIS</a>) with funding provided by NASA/HQ.",
+            "html_attribution": "Imagery provided by services from the Global Imagery Browse Services (GIBS), operated by the NASA/GSFC/Earth Science Data and Information System (<a href=\"https://earthdata.nasa.gov\">ESDIS</a>) with funding provided by NASA/HQ.",
+            "name": "NASAGIBS.ModisTerraBands721CR"
```

```suggestion
            "name": "NASAGIBS.ModisTerraBands721CR",
            "time": ""
```
If there's a `time` placeholder we need to define an attribute for that.

---

done